Public/Src/Utilities/KeyValueStore/KeyValueStoreAccessor.cs

```diff
@@ -642,6 +642,9 @@ public static IEnumerable<string> ListColumnFamilies(string storeDirectory)
         }
 
         /// <inheritdoc />
+        /// <summary>
+        /// Finishes up/cleans remaining RocksDB tasks and flushes DB to disk.
+        /// </summary>

```

Comment:
pick one - either inheritdoc OR summary

Reply:
Gotcha, ill go with the summary as that is more relevant. 

---
In reply to: [306044653](https://github.com/microsoft/BuildXL/pull/636#discussion_r306044653) [](ancestors = 306044653)